lib/crutches/keyword.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Crutches.Keyword do
+  @doc ~S"""
+  Checks a keyword list for unexpected keys, by using a default list of keys. When a bad key is detected it returns false, else it returns true.
```

Comment:
Can we break up this line so it's under 80 chars (and on line 22)? I know the issue didn't do this, unfortunately Markdown was giving me a hassle. I apologize for that.